redfish-core/include/utils/hw_isolation.hpp

```diff
@@ -187,10 +187,14 @@ inline void
 {
     // Get the HardwareIsolation entry by using the given resource
     // associations endpoints
-    crow::connections::systemBus->async_method_call(
-        [aResp, resourceObjPath, hwIsolationDbusName](
-            boost::system::error_code ec,
-            const std::variant<std::vector<std::string>>& vEndpoints) {
+
+    sdbusplus::asio::getProperty<std::vector<std::string>>(
+        *crow::connections::systemBus, "xyz.openbmc_project.ObjectMapper",
+        resourceObjPath.str + "/isolated_hw_entry",
+        "xyz.openbmc_project.Association", "endpoints",
+        [aResp, resourceObjPath,
+         hwIsolationDbusName](boost::system::error_code ec,
+                              const std::vector<std::string>& vEndpoints) {
```

Comment:
I think we can use getAssociationEndPoints() like

```
    dbus::utility::getAssociationEndPoints(
        resourceObjPath.str + "/isolated_hw_entry",
        [asyncResp, pcieSlotPath,
         [aResp, resourceObjPath,
         hwIsolationDbusName](boost::system::error_code ec,
                const dbus::utility::MapperEndPoints& vEndpoints) {
```

Reply:
Thank you! I addressed it.